src/global_styling/mixins/_shadow.scss

```diff
@@ -12,19 +12,19 @@
 // This file uses RGBA literal values responsibly
 // This file uses off-pattern indentation to be more readable
 
-// sass-lint:disable no-color-literals, no-color-keywords, indentation, quotes
+// sass-lint:disable no-color-literals, no-color-keywords, indentation, quotes, hex-length hex-notation leading-zero
```

Comment:
I'm wondering if there was really a need to disable "hex-length", "hex-notation", "leading-zero"? I've also noticed this in other files as well

Reply:
They would trigger `sass-lint` errors if not muted